pkg/config/agentmanagement.go

```diff
@@ -15,7 +16,11 @@ import (
 	"github.com/prometheus/common/config"
 )
 
-const cacheFilename = "remote-config-cache.yaml"
+const (
+	cacheFilename     = "remote-config-cache.yaml"
+	httpMinJitterTime = time.Duration(0)
+	httpMaxJitterTime = 10 * time.Second
```

Comment:
What are those values coming from? I'm not sure how jitter implementation should be but I thought that it would be related with the polling interval. How we ensure that requests are somehow equally distributed when we have big polling intervals (such 60s)?